src/components/Proposal/ProposalDetailsPage.tsx

```diff
@@ -175,7 +175,7 @@ class ProposalDetailsPage extends React.Component<IProps, IState> {
                     <ProposalCountdown proposal={proposal} detailView /> :
                     <span className={css.closedTime}>
                       {proposal.stage === IProposalStage.Queued ? "Expired" :
-                        proposal.stage === IProposalStage.PreBoosted ? "Ready to Boost" :
+                        proposal.stage === IProposalStage.PreBoosted ? "Ready to Unboost" :
```

Comment:
the table should be or  "Ready to Boost"  or "Ready to Unboost"  according to the proposal state

Reply:
@Razzwan ?